operations/helm/charts/grafana-agent/values.yaml

```diff
@@ -103,6 +103,8 @@ configReloader:
     tag: v0.8.0
   # -- Override the args passed to the container.
   customArgs: []
+  # -- Resource requests and limits to apply to the config reloader container.
+  resources: {}
```

Comment:
Should we provide some defaults here? We should roughly know what the requests should be, and don't expect the load to ever change.

Reply:
Good point; I've added some extra-small defaults to make HPA provisioning easier.